.github/workflows/integration-test.yml

```diff
@@ -8,11 +8,22 @@ on:
 
 jobs:
 
-  build:
-
+  integration-tests:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        build_type: [local-data, remote-data]
 
     steps:
     - uses: actions/checkout@v2
+    - name: Download data
+      env:
+        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        AWS_DEFAULT_REGION: ${{ secrets.AWS_DEFAULT_REGION }}
+        REMOTE_DATA_URL: ${{ secrets.REMOTE_DATA_URL }}
+        REMOTE_CONFIGURATION_URL: ${{ secrets.REMOTE_CONFIGURATION_URL }}
+      if: ${{ matrix.build_type == 'remote-data' && env.REMOTE_DATA_URL != '' }}
```

Comment:
however we only run this job if we have a URL for the remote data. Otherwise it gets skipped. Which for the INL repo is probably the right thing to do until enabling remote data download/test